test/e2e/all_in_one_test.go

```diff
@@ -238,11 +238,9 @@ func TestDeployAllInOnePostgres(t *testing.T) {
 	t.Log("building test cluster and environment")
 	addons := []clusters.Addon{}
 	addons = append(addons, metallb.New())
-	if imageLoad != "" {
-		b, err := loadimage.NewBuilder().WithImage(imageLoad)
```

Comment:
I'm just wondering if we wouldn't extend the feature in KTF so that we can use it here instead of adding glue code in KIC. WDYT?

Reply:
Yes, I also considered to update the `loadimage` addon to load multiple images in KTF, but this needs to implement in KTF first, then release KTF, at last we can bump version of KTF and update KIC. This is quit a long process.

Reply:
👍 Agreed. Do you mind creating an issue for this (with reference to this PR) so that we don't loose it?

Reply:
https://github.com/Kong/kubernetes-testing-framework/issues/366